db/models/api.model.js

```diff
@@ -8,10 +8,30 @@ function readTopics() {
 }
 
 function readArticleId(articleId) {
-  // console.log(articleId)
+
+  const query = `
+    SELECT 
+      articles.article_id,
+      articles.title,
+      articles.body,
+      articles.topic,
+      articles.author,
+      articles.created_at,
+      articles.votes,
+      articles.article_img_url,
+      COUNT(comments.article_id) AS comment_count
```

Comment:
Good job getting the COUNT in here - you could also have a look at casting to see if you can change the data type to a number 👀